### ISSUE TITLE
Train/Test/Val Tensorflow Model

### ISSUE THREAD

**Issue by Ivaldivi** [Ivaldivi/DATA515-Image-Classification#30]
Try train/test/val split of train data: 
x_train, x_test, y_train, y_test = train_test_split(
    washington_images_stack, encoded_labels, test_size=0.2, random_state=42
)

x_train, x_val, y_train, y_val = train_test_split(
    x_train, y_train, test_size=0.2, random_state=42
) # 0.16 = validation set

* Re-train using new split
* Look at testing data output

**Comment by anthony-nguyen-04** [comment]
https://github.com/Ivaldivi/DATA515-Image-Classification/pull/49

Train/Test/Val split done
